src/test/java/com/tmartsyniak/learning/shapes/task1/CircleTest.java

```diff
@@ -0,0 +1,39 @@
+package com.tmartsyniak.learning.shapes.task1;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class CircleTest {
+
+    private double radius;
+    private Circle circle;
+    private double radiusWrong;
+    Circle circleWrong;
+
+    @Before
+    public void setUp() throws Exception {
+        radius = 5;
```

Comment:
It's better to put this in test local variable. Imagine that you want to write one test which needs circle with radius 5, and the other one with radius 6. You won't be able to this your approach.